cli/ballerina-launcher/src/main/java/org/ballerinalang/launcher/Main.java

```diff
@@ -415,6 +424,74 @@ public void setSelfCmdParser(JCommander selfCmdParser) {
         }
     }
 
+    /**
+     * Represents the encrypt command which can be used to make use of the AES cipher tool. This is for the users to be
+     * able to encrypt sensitive values before adding them to config files.
+     *
+     * @since 0.965.0
+     */
+    public static class EncryptCmd implements BLauncherCmd {
+
+        @Parameter(names = "--java.debug", hidden = true)
+        private String javaDebugPort;
+
+        @Parameter(names = {"--help", "-h"}, hidden = true)
+        private boolean helpFlag;
+
+        private JCommander parentCmdParser;
+
+        @Override
+        public void execute() {
+            if (helpFlag) {
+                String commandUsageInfo = BLauncherCmd.getCommandUsageInfo(parentCmdParser, "encrypt");
+                outStream.println(commandUsageInfo);
+                return;
+            }
+
+            outStream.println("ballerina: enter the value to be encrypted:");
+            String value = new String(System.console().readPassword());
+
+            outStream.println("ballerina: enter a secret to be used in encrypting the value:");
```

Comment:
This is how OpenSSL works: 
> enter aes-256-cbc encryption password:
> Verifying - enter aes-256-cbc encryption password:

Basically, it asks for password verification, before encrypting the value with (possibly misspelled) password. This is same with keytool passwords, etc. IMO, it is better to to the same.

Reply:
ack.